client/src/components/mapIntroduction.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+// @ts-ignore
+import lightbulbIcon from '/node_modules/uswds/dist/img/usa-icons/lightbulb_outline.svg';
+import * as styles from './mapIntroduction.module.scss';
+
+const MapIntroduction = () => {
+  return (
+    <aside className={styles.mapIntroContainer}>
+      <header className={styles.mapIntroHeader}>Zoom and select a census block group to view data</header>
```

Comment:
We also want to localize this text - you can see examples here: https://github.com/usds/justice40-tool/blob/7b241795fa9c7f70370b6913cc7dbc6cf57209c4/client/src/components/territoryFocusControl.tsx and here: https://github.com/usds/justice40-tool/blob/cfce0dc8269be08c277edbda339d66e65b703cdf/client/src/components/J40Header.tsx

Reply:
added intl!